### DISCUSSION TITLE
Updating grunge-stack to lastest versions

### DISCUSSION THREAD

**Discussion by jarimustonen** [remix-run/remix#8736] [General]
As the software evolves, the stack should be such that is could be updated to latest version. To this end, I suggest that grunge-stack is updated to latest versions.

I have tried this manually and was able to fix a few errors:

* A few packages must be added explicitly (`@smithy/node-http-handler`, `@aws-sdk/lib-dynamodb`, and `@aws-sdk/client-dynamodb`)
* `isbot` should be retained at the version `3.8.0` due to breaking changes in the version `4.0.0`
* `remix.config.js` must be added the following configuration: `browserNodeBuiltinsPolyfill: { modules: { querystring: true } }`

Finally there is also an error related to cypress tests. The tests fail with:

```
Lambda init error: The requested module 'msw' does not provide an export named 'rest'
http://localhost:8811 timed out on retry 9 of 3, elapsed 95619ms, limit 90000ms
Error: Response code 500 (Internal Server Error)
```

I'm at lost here and really do not know what to do with this. If someone can help with this one, I could make pull request. Or maybe the Remix team would like to update the stack,

**Comment by MichaelDeBoey** [comment]
Please create an issue in the Grunge stack repo for this

**Comment by jarimustonen** [comment]
The corresponding issue: https://github.com/remix-run/grunge-stack/issues/185
